### ISSUE TITLE
ServicePulse shows incomplete page and gives Angular module loading error (Tested 1.3.3 and 1.3.4)

### ISSUE THREAD

**Issue by trygvelo** [Particular/ServicePulse#211]
I upgraded to 1.3.3 yesterday (from 1.2.1 IIRC) and got this error. Didn't have time to investigate or file new issue then, and discovered the new 1.3.4 version today. However, 1.3.4 gives the same behaviour. Running on Firefox (latest), win server 2012. (Same thing on IE 11 too).

![image](https://cloud.githubusercontent.com/assets/1652052/10969815/e6359122-83ca-11e5-9d9e-ebde11c8b0ca.png)

**Comment by petersgiles** [comment]
I have found that Firefox has a very aggressive cache.

Try this go to your FireFox options
![firefox options](https://cloud.githubusercontent.com/assets/1644481/10983644/ea810994-8467-11e5-82d4-b2b4ae8a7b8a.PNG)

Then go to advanced and clear the cache

![firefox clear cache](https://cloud.githubusercontent.com/assets/1644481/10983648/edc4433c-8467-11e5-8b99-9ede564fb619.PNG)



If that isn't making a difference then go the the ServicePulse install directory (on my machine it's C:\Program Files (x86)\Particular Software\ServicePulse). Just for safety make a backup of the app directory and put it somewhere safe. You probably won't need it but it doesn't hurt to be cautious.

Look in the app folder it should look like this

![servicepulse clear](https://cloud.githubusercontent.com/assets/1644481/10983745/86705a8a-8468-11e5-95d5-ac5876b9952b.PNG)

and the js directory should look like this

![servicepulse clear js](https://cloud.githubusercontent.com/assets/1644481/10983747/8884b622-8468-11e5-8395-ff0e3a64b79b.PNG)

If there are extra files in there remove them. Your problem is likely to be caused by app.js in the app folder. 

Clear the browser cache and try again.

If that doesn't make a difference. Uninstall ServicePulse, delete C:\Program Files (x86)\Particular Software\ServicePulse and then reinstall.

The installer is meant to take care of these extra files and I tested several upgrade scenarios so I'm interested to know which of these solutions (if any) fixes your problem.
